src/vmspawn/vmspawn-util.c

```diff
@@ -0,0 +1,341 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "architecture.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "json.h"
+#include "log.h"
+#include "macro-fundamental.h"
+#include "memory-util.h"
+#include "path-lookup.h"
+#include "path-util.h"
+#include "sort-util.h"
+#include "string-util-fundamental.h"
+#include "string-util.h"
+#include "recurse-dir.h"
+#include "strv.h"
+#include "vmspawn-util.h"
+#include <stdio.h>
+
+OvmfConfig* ovmf_config_new(void) {
+        return new0(OvmfConfig, 1);
+}
+
+OvmfConfig* ovmf_config_free(OvmfConfig *config) {
+        config->path = mfree(config->path);
+        config->vars = mfree(config->vars);
```

Comment:
There's no need to assign these to `NULL`. The object itself is `free`-ed after all.

Reply:
I normally assign to NULL after freeing an object to guard against use-after-free vulnerabilities, though I guess its less important as its freed immediately after. I'll just call free on these instead.

Reply:
Fixed in f8c2832218.